### ISSUE TITLE
Key Abduction Didn't Forge a Key

### ISSUE THREAD

**Issue by wistro** [jeremylarner/keyteki#32]
See attached screenshot. I had 8 aember after playing the card and 7 cards in hand which meant a key should have been forged for 8 aember (6+9-7) but nothing happened.

<img width="1680" alt="screen shot 2018-11-23 at 8 01 14 pm" src="https://user-images.githubusercontent.com/28895320/48959045-c706b200-ef5a-11e8-9afa-002573373a6e.png">

**Comment by wistro** [comment]
I've tested this card some more and it looks like I've found the bug:

It doesn't prompt you to forge a key if no Mars creatures are picked up into hand by it's first line. But that's irrelevant to whether or not you're allowed to forge a key in the second half of the card text. "Return each Mars creature to its owner's hand" can return 0 Mars creatures without issue and I should still be able to forge a key if I have enough aember to meet the cost.
